clean_project/search.py

```diff
@@ -0,0 +1,60 @@
+"""
+Search a word in a file and return all the occurrence specified the line numbers
+"""
+import argparse
+import psycopg2
+
+BOLD = '\033[1m'
+END = '\033[0m'
+
+
+def search(search_word):
+    """
+    Search if a given word is in bucket of files
+    :param search_word: Word given to search
+    :return: List of files and numbers of file when search_word occurrence
+    """
+    connection = None
+    result_search = []
+    try:
+        connection = psycopg2.connect(database='index_db', user='gianluca', password='password')
```

Comment:
hmmm there are a few solutions, although on a second thought it's probably actually better to leave this as you did it.

nevertheless, the ideal design should include a class with a constructor you can pass the connection parameters to. in the search method then you can instantiate the connection without having to embed the details into